packages/garbo/src/embezzler/fights.ts

```diff
@@ -411,29 +419,29 @@ export const copySources = [
       );
     },
   ),
-  new EmbezzlerFight(
+  new CopyTargetFight(
     "Green Taffy",
     () =>
       have($item`envyfish egg`) &&
-      get("envyfishMonster") === embezzler &&
+      get("envyfishMonster") === globalOptions.target &&
       !get("_envyfishEggUsed"),
     () =>
       have($item`envyfish egg`) &&
-      get("envyfishMonster") === embezzler &&
+      get("envyfishMonster") === globalOptions.target &&
       !get("_envyfishEggUsed")
         ? 1
         : 0,
     (options: RunOptions) => {
       withMacro(options.macro, () => use($item`envyfish egg`)), options.useAuto;
     },
   ),
-  new EmbezzlerFight(
+  new CopyTargetFight(
     "Screencapped Monster",
     () =>
       have($item`screencapped monster`) &&
-      property.getString("screencappedMonster") === "Knob Goblin Embezzler",
+      property.get("screencappedMonster") === globalOptions.target,
```

Comment:
most of the `getString`s are prefs that get set to weird, inappropriate values when not set (e.g., olfaction used to get set to "wabbit"). If you're switching them to `get`s, be certain that it's safe to do so, and also just import `get` directly instead of doing `propert.get`

Reply:
Is it? Ugh. We should maybe normalise that on the libram side.

Reply:
I cannot confirm that this is still the case, but it was the case when this was written.